dll/ntdll/CMakeLists.txt

```diff
@@ -61,7 +61,7 @@ set_module_type(ntdll win32dll ENTRYPOINT 0)
 set_subsystem(ntdll console)
 ################# END  HACK #################
 
-target_link_libraries(ntdll csrlib rtl rtl_um rtl_vista ntdllsys libcntpr uuid ${PSEH_LIB})
+target_link_libraries(ntdll csrlib rtl rtl_um rtl_vista ntdllsys libcntpr uuid cryptlib ${PSEH_LIB})
```

Comment:
Is the cryptlib addition only for NT6+?
If so, you could something like:
```
if(DLL_EXPORT_VERSION GREATER_EQUAL 0x600)
    target_link_libraries(ntdll cryptlib)
endif()
```
in addition to the original (untouched) target_link_libraries line.

Reply:
Yes, committed.